prisma/seed.js

```diff
@@ -30,41 +30,216 @@ async function seed() {
     }
   })
 
+  const thirdUser = await prisma.user.create({
+    data: {
+      email: 'third@blah',
+      password,
+      cohortId: cohort1.id
+    }
+  })
+
+  const fourthUser = await prisma.user.create({
+    data: {
+      email: 'fourth@blah',
+      password,
+      cohortId: cohort2.id
+    }
+  })
+
+  const fifthUser = await prisma.user.create({
+    data: {
+      email: 'fifth@blah',
+      password,
+      cohortId: cohort1.id
+    }
+  })
+
+  const sixthUser = await prisma.user.create({
+    data: {
+      email: 'sixth@blah',
+      password,
+      cohortId: cohort2.id
+    }
+  })
+
+  const seventhUser = await prisma.user.create({
+    data: {
+      email: 'seventh@blah',
+      password,
+      cohortId: cohort1.id
+    }
+  })
+
+  const eightUser = await prisma.user.create({
+    data: {
+      email: 'eight@blah',
+      password,
+      cohortId: cohort2.id
+    }
+  })
+
+  const ninthUser = await prisma.user.create({
+    data: {
+      email: 'ninth@blah',
+      password,
+      cohortId: cohort1.id
+    }
+  })
+
+  const tenthUser = await prisma.user.create({
```

Comment:
That's a lot of repetition!

There are two ways I think you could really improve this.

1. Use Prisma's `createMany` function to create 10 users. You can run a `findMany` afterwards to get all of their ID's - though this still has the problem of having a lot of lines of code where you're defining different email addresses

2. (Preferable) Use a loop to `create` 10 users, their profiles and their posts in one query, using the index to create unique email addresses. You can save each created user (include their profile and posts in the returned data from prisma) into an array during the loop, then run another loop to create some likes.

What do you think?

Reply:
Yeah I think I will go for the second option, I will improve this and get back to you when it is done!!!